Source/Utilities/DOMReady.js

```diff
@@ -77,6 +77,7 @@ if ('onreadystatechange' in document) document.addListener('readystatechange', c
 else shouldPoll = true;
 
 if (shouldPoll) poll();
+if (!ready) check();
```

Comment:
Which check does it actually execute? The `readyState` or the `scroll` test. The last one isn't included in the minus-ie8 build.